### DISCUSSION TITLE
Option to automatically toggle when using system/browser dark theme.

### DISCUSSION THREAD

**Discussion by d-muis** [darkreader/darkreader#8837] [Ideas]
Browsers can be set to follow the system dark theme, which a lot of people set up to automatically change depending on the time of day. It would be a great addition, if this extension had a setting to follow either the system or browser theme and automatically toggle when it is changed. It would greatly complement OS-wide dark-theme switching on all platforms.

Apologies if this has been mentioned before.

EDIT: After a little bit of digging, I found out that this option already exists in the "New Design" accessible through the "Dev Tools" menu, thanks a lot for adding it, along with automatic dark mode detection, it works perfectly!

**Comment by LittleFreak** [comment]
+1
